lib/src/offline_region.dart

```diff
@@ -33,11 +33,7 @@ class OfflineRegionDefinition {
 
   factory OfflineRegionDefinition.fromMap(Map<String, dynamic> map) {
     return OfflineRegionDefinition(
-      bounds: map['bounds'] != null
-          ? _latLngBoundsFromList(
-              map['bounds'],
-            )
-          : null,
+      bounds: _latLngBoundsFromList(map['bounds']),
```

Comment:
Are we relying on the platform specific code to reliably provide `map['bounds']` here? I think that's ok, just confirming.

Reply:
Yes. Neither iOS or Android can return `null` from the platform side.